Command/InitCommand.php

```diff
@@ -41,7 +43,7 @@ public function configure()
         $this->setHelp('This command will simply call the initialize method of the currently active search adapter.');
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
```

Comment:
small BC Break in all commands, but think the commands execute method should not be overridden.